Sources/tart/Commands/IP.swift

```diff
@@ -34,10 +34,20 @@ struct IP: AsyncParsableCommand {
     let vmConfig = try VMConfig.init(fromURL: vmDir.configURL)
     let vmMACAddress = MACAddress(fromString: vmConfig.macAddress.string)!
 
-    guard let ip = try await IP.resolveIP(vmMACAddress, resolutionStrategy: resolver, secondsToWait: wait) else {
+    let lock = try PIDLock(lockURL: vmDir.configURL)
+    if try lock.pid() == 0 {
       throw RuntimeError.NoIPAddressFound("no IP address found, is your VM running?")
     }
-    print(ip)
+
+    let ip = try await IP.resolveIP(vmMACAddress, resolutionStrategy: resolver, secondsToWait: wait)
```

Comment:
Is there a reason to stop using `guard` here (as it was before)?

You can avoid things like `ip == nil` and `ip!` below if you'll keep using it.